### PR TITLE
FIX-TEMPLATE: 201-dynamic-web-tests

### PR DESCRIPTION
**Pull request by olga-spektra**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

File: 201-dynamic-web-tests/azuredeploy.parameters.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```
Comment (bmoore-msft) [inline 201-dynamic-web-tests/azuredeploy.parameters.json:2]: revert this change

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: 201-dynamic-web-tests/azuredeploy.json [hunk 2]
```diff
@@ -1,11 +1,45 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```
[the remark this answers is not in the document]
Comment (olga-spektra) [reply 201-dynamic-web-tests/azuredeploy.json]: done!

File: 201-dynamic-web-tests/azuredeploy.parameters.json (hunk 1 above)
Comment (olga-spektra) [reply 201-dynamic-web-tests/azuredeploy.parameters.json]: done!
